webknossos/webknossos/dataset/_utils/buffered_slice_writer.py

```diff
@@ -124,12 +129,22 @@ def _write_buffer(self) -> None:
             buffer_start_list[self.dimension] = self.buffer_start_slice
             buffer_start = Vec3Int(buffer_start_list)
             buffer_start_mag1 = buffer_start * self.view.mag.to_vec3_int()
-            self.view.write(
-                data,
-                offset=buffer_start.add_or_none(self.offset),
-                relative_offset=buffer_start_mag1.add_or_none(self.relative_offset),
-                absolute_offset=buffer_start_mag1.add_or_none(self.absolute_offset),
-            )
+            if isinstance(self.view, MagView):
+                self.view.write(
+                    data,
+                    offset=buffer_start.add_or_none(self.offset),
+                    relative_offset=buffer_start_mag1.add_or_none(self.relative_offset),
+                    absolute_offset=buffer_start_mag1.add_or_none(self.absolute_offset),
+                    update_bbox=self.update_bbox,
+                )
+            else:
+                self.view.write(
+                    data,
+                    offset=buffer_start.add_or_none(self.offset),
+                    relative_offset=buffer_start_mag1.add_or_none(self.relative_offset),
+                    absolute_offset=buffer_start_mag1.add_or_none(self.absolute_offset),
+                    allow_write_outside_bbox=not self.update_bbox,
```

Comment:
I don't understand this logic. If the bbox must not be updated, writing outside of the bbox is allowed? Shouldn't this be inverted?